certified-connectors/Office 365 Users/README.md

```diff
@@ -0,0 +1,49 @@
+# Office 365 Users Connector
+
+The connector uses Microsoft Graph to perform operations on Office 365 account users and their relationships with other users. It aims to enable actions such as getting and updating the signed-in user profile, listing users within the organizations, getting user's manager and direct reports, listing the trending documents, etc.
+
+## Pre-requisites
+
+You will need the following to proceed:
+
+- A Microsoft Power Apps or Microsoft Power Automate plan with custom connector feature
+- An Azure subscription
```

Comment:
Do customers need azure subscription for this?

Reply:
It is needed to create Azure AD client application.